Source/JavaScriptCore/wasm/WasmTable.cpp

```diff
@@ -263,28 +293,23 @@ FuncRefTable::FuncRefTable(uint32_t initial, std::optional<uint32_t> maximum, Ty
     // FIXME: It might be worth trying to pre-allocate maximum here. The spec recommends doing so.
     // But for now, we're not doing that.
     // FIXME this over-allocates and could be smarter about not committing all of that memory https://bugs.webkit.org/show_bug.cgi?id=181425
+    uint32_t actualLength = allocatedLength(m_length);
```

Comment:
The fact that allocatedLength is `roundUpToPowerOfTwo(length)` seems kinda insane but we can fix that in a different patch.